app/models/concerns/pageflow/hosted_file.rb

```diff
@@ -62,27 +70,6 @@ def original_url
       url
     end
 
-    # @deprecated Write a migration instead
-    def self.columns(t)
-      t.belongs_to(:entry, index: true)
-      t.belongs_to(:uploader, index: true)
-
-      t.string(:state)
-      t.string(:rights)
-
-      t.string(:attachment_on_filesystem_file_name)
-      t.string(:attachment_on_filesystem_content_type)
-      t.integer(:attachment_on_filesystem_file_size, limit: 8)
-      t.datetime(:attachment_on_filesystem_updated_at)
-
-      t.string(:attachment_file_name)
-      t.string(:attachment_content_type)
-      t.integer(:attachment_file_size, limit: 8)
-      t.datetime(:attachment_updated_at)
-
-      t.timestamps
-    end
-
```

Comment:
Extracted this into #1085 since it's a breaking change that needs to go into 13.0 and this PR will probably be part of 13.1. Can you rebase onto master?

Reply:
Done